exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,162 @@
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class PokerTest
+{
```

Comment:
No problem, that's completely understandable! We all have other committments as well so it's fine to need some time before answering comments and updating reviews. This is meant to be fun to do so don't feel pressured to do everything really fast if you're busy. It's better to do it well and have fun while doing it, than doing it super fast :)

According to [this page](http://stackoverflow.com/questions/15655126/how-to-auto-format-code-in-eclipse) the shortcut on eclipse is ctrl + shift + F :)